### ISSUE TITLE
[BUG] Immich logo on Shared page takes to immich.app instead 

### ISSUE THREAD

**Issue by anuj-online** [immich-app/immich#5519]
### The bug

When a shared album and a link is created (password protected)
Upon click the Immich logo on the top left, the page redirects to immich.app instead of the host url

### The OS that Immich Server is running on

Ubuntu 22.10

### Version of Immich Server

v1.89.0

### Version of Immich Mobile App

v1.89.0 build 129

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
NA
```

### Your .env content

```Shell
NA
```

### Reproduction steps

```bash
1. Run immich and create a share album
2. Share the link with password to other user
3. When user enters the password and previews the gallery (all is fine)
4. Upon clicking the immich logo on top right, the page goes to immich.app, instead of the server url the link is shared from
```

**Comment by jrasm91** [comment]
"its a feature not a bug"

**Comment by Luvbe3** [comment]
I think clicking on the icon will redirect you to the shared homepages or the homepages of the web, which is more reasonable than switching to immich.app

**Comment by arpanghosh8453** [comment]
Thank you for fixing this. I am waiting for the docker version release!

**Comment by arpanghosh8453** [comment]
I am not sure if I am missing something, I updated to v1.91.0 and restarted my containers, it pulled down the new builds and all, but still I see the same issue with shared links. Any clue what I am doing wrong?

**Comment by jrasm91** [comment]
I think the PR only fixed it for shared albums not shared individual assets.

**Comment by JW-CH** [comment]
I will have a look and fix it for the shared assets aswell.

**Comment by Luvbe3** [comment]
The problem still exists in INDIVIDUAL SHARE

![Untitled](https://github.com/immich-app/immich/assets/153334206/496473b5-9faa-470c-85ea-c4fb49571671)

**Comment by danieldietzler** [comment]
Hey, this has been fixed #5889 and will be in the next release :)
